src/app_service/service/settings/service.nim

```diff
@@ -1103,7 +1105,13 @@ QtObject:
     if self.settings.backupPath == value:
       return
     try:
-      let formattedPath = singletonInstance.utils.fromPathUri(value)
+      var formattedPath = value
```

Comment:
Do we need the same on `write` to make sure it succeeds ? Not 100%, but I think the permissions could change. On my device for example the app permissions are dropped if I don't use the app for some time.